### PR TITLE
Week 6 Synthesizing Task

### PR DESCRIPTION
**Pull request by eliz79**
Week 6 Synthesizing Task.

### CODE REVIEW

File: task_01.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        None
+
+        >>> task_01.get_current_date()
+        datetime.date(2015, 3, 5)
+        $ python -i task_01.py
+        2015-03-05
+    """
+    return datetime.date.today()
+
+CURDATE = get_current_date()
```
Comment (cheuschober) [inline task_01.py]: @eliz79 Good so-far but right now CURDATE is executing EVERY time -- we don't want it to execute when the file is imported (we want it to remain `None`). Review the Module Execution video on how to do this.

Comment (eliz79) [inline task_01.py]: Professor, 

So I added:
if __name__ == '__main__':

but not sure if i need to create a new pull request.

File: task_01.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+if __name__ == '__main__':
```
Comment (cheuschober) [inline task_01.py]: @eliz79 If you'd had line 32 inside the block at line 34 you would have aced this!

**Comment by eliz79** [comment]
I didn't have the option of resubmit, so i posed it on the Blackboard DB section. Otherwise here it is as well: 
https://github.com/is210-2015-spring-01/is210-week-06-synthesizing/pull/17

**Pull request merged**